src/modules/direction/store/directionSlice.ts

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-param-reassign */
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import MOCK_EXPERTS from '../../main/mockDataExperts';
+import { IExpert } from '../../../lib/types';
+import type { AppThunkType } from '../../../store/store';
+
+export interface IDirectionState {
+  experts: IExpert[];
+}
+
+const initialState: IDirectionState = {
+  experts: [],
+};
+
+export const directionSlice = createSlice({
+  name: 'direction',
+  initialState,
+  reducers: {
+    loadExperts: (state, action: PayloadAction<IExpert[]>) => {
+      state.experts = action.payload;
+    },
```

Comment:
```suggestion
    loadExperts: (state, action: PayloadAction<IExpert[]>) =>  state.experts = action.payload,
```
you can do it in one line :)

Reply:
eslint say - no I can't